src/main/java/com/adobe/abp/regola/rules/NumberRule.java

```diff
@@ -58,11 +58,30 @@ Result check(V fact) {
                 return checkFact(fact, (f, value) -> comparator.compare(f, getValue()) < 0);
             case LESS_THAN_EQUAL:
                 return checkFact(fact, (f, value) -> comparator.compare(f, getValue()) <= 0);
+            case DIVISIBLE_BY:
+                if (!(fact instanceof Integer || fact instanceof Long) ||
```

Comment:
Not all branches are tested.

![Screenshot 2024-11-25 at 17 15 20](https://github.com/user-attachments/assets/a44e75d9-7098-46d5-8ec0-8aeb63423c5d)